solutions/only_ints.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for verifying integer inputs.
+
+Module contents:
+    - only_ints: A function to check if two parameters are both integers.
+
+Created on 12 29 2024
+@author: Mohammad
+"""
+
+
+def only_ints(param1, param2):
+    """
+    Using this function, you will be able to take two parameters, and it will return True
+    if both parameters are integers, and otherwise it is false
+
+    Parameters:
+    param1 (int): The first parameter to check, expected to be an integer.
+    param2 (int): The second parameter to check, expected to be an integer.
+
+    Returns:
+    bool: True if both parameters are integers, False otherwise.
+
+    Raises:
+    TypeError: If any of the parameters is not an integer.
+    """
+    if not isinstance(param1, int) or not isinstance(param2, int):
```

Comment:
what about putting "positive" if case? like 
if isinstance(param1, int) and isinstance(param2, int): #I may be mistaken with syntax

Reply:
Oh I almost forgot about it, I can make it make it more clear